src/commands/android/subcommands/connect/index.ts

```diff
@@ -1,7 +1,15 @@
+import {verifyOptions} from '../common';
 import {connectWirelessAdb} from './wireless';
 import {Options, Platform} from '../../interfaces';
 
 export async function connect(options: Options, sdkRoot: string, platform: Platform): Promise<boolean> {
+  const optionsPassed = Object.keys(options).filter(option => options[option] === true);
+
```

Comment:
Since this line is something that would be duplicated in all the subcommand index files, it should also be put inside the `verifyOptions` itself. Let's say, if we want to change something in the logic behind this line in the future (like also include options other than the boolean options), we shouldn't need to change it at multiple places.